wire/modules/Process/ProcessModule/ProcessModule.module

```diff
@@ -206,6 +215,11 @@ class ProcessModule extends Process {
 				// check that it can be installed NOW (i.e. all dependencies met)
 				if(!$this->wire('modules')->isInstallable($moduleName, true)) continue; 
 			}
+
+			if($translated) {
+				// exclude modules without translations present
+				if(!$info['languages'] || !is_array($info['languages']) || !count($info['languages'])) continue;
```

Comment:
Note: `!count($info['languages'])` is unnecessary since an empty array won't pass `!$info['languages']`. PHP converts an empty array to boolean `FALSE` and an array with items to boolean `TRUE`.

On the other hand you should first check if `isset($info['languages'])` or you might run into an `E_NOTICE` :)